### ISSUE TITLE
Plural name and label missing for "epoch" in vocab/properties.json

### ISSUE THREAD

**Issue by ehennestad** [openMetadataInitiative/openMINDS#21]
Is this intentional:
https://github.com/openMetadataInitiative/openMINDS/blob/b34dbefa74e218fca7aff9df71075d1f4b5215db/vocab/properties.json#L4554C1-L4572C5

**Comment by lzehl** [comment]
no, this new property should at least list all attributes that we expect to be filled. so something went wrong in the pipeline. we need to investigate. thanks for bringing this to our attention

**Comment by lzehl** [comment]
@Raphael-Gazzotti and me discovered that there is a larger build issue for the vocab. For "epoch" it seems that v3.0 is ignored. In addition the default attributes need to be set up correctly. This needs further investigation

**Comment by lzehl** [comment]
v3.0 appearance is fixed with https://github.com/openMetadataInitiative/openMINDS/pull/42
